Makefile

```diff
@@ -1,6 +1,5 @@
-PREFIX=docker-machine-driver-kvm
-MACHINE_VERSION=v0.10.0
-GO_VERSION=1.8.1
+PREFIX=crc-driver-libvirt
```

Comment:
this is not a prefix, but `COMMAND=crc-driver-libvirt`
a `PREFIX` is something like `/usr/local` without specifying `/bin/`, so it can be either `/bin/`, `/usr/bin/`, `/usr/local/bin/`, `~/.local/bin/`, etc`